src/substrait/builders/extended_expression.py

```diff
@@ -240,7 +240,7 @@ def resolve(
                 extension_function=ste.SimpleExtensionDeclaration.ExtensionFunction(
                     extension_uri_reference=registry.lookup_uri(uri),
                     function_anchor=func[0].anchor,
-                    name=function,
+                    name=str(func[0]),
```

Comment:
I think you should do the same for window_function and aggregate_function as well

Reply:
well consistent at least.... the web pages don't spec what happen with no arguments 🤷